pyproject.toml

```diff
@@ -64,13 +77,25 @@ disable = [
 ]
 
 [tool.black]
-line-length = 100
-target-version = ["py37"]
+line-length = 79
```

Comment:
Can you revert this please? Curious on the additional config for include and exclude as black has pretty sensible defaults here. Did you find it to be problematic?

Reply:
Yes! My org uses different line-lengths, thats why :)